tests/unit/data/fpdsXML.txt

```diff
@@ -200,12 +200,21 @@
               <ns1:congressionalDistrictCode>08</ns1:congressionalDistrictCode></ns1:vendorLocation>
             <ns1:vendorSiteCode>806585717</ns1:vendorSiteCode>
             <ns1:vendorAlternateSiteCode>22202</ns1:vendorAlternateSiteCode>
-            <ns1:vendorDUNSInformation>
-              <ns1:DUNSNumber>806585717</ns1:DUNSNumber>
-              <ns1:vendorName>B.F. SAUL PROPERTY COMPANY</ns1:vendorName>
-              <ns1:globalParentDUNSNumber>006920342</ns1:globalParentDUNSNumber>
-              <ns1:globalParentDUNSName>B.F. SAUL COMPANY</ns1:globalParentDUNSName>
-              <ns1:cageCode>0PW76</ns1:cageCode></ns1:vendorDUNSInformation>
+            <ns1:entityIdentifiers>
```

Comment:
Can the test that uses this file reflect the changes?

Reply:
Update: This would need to be a massive tech debt to update the test to include the various fields it's importing. This is fine for now.